code/modules/projectiles/projectile/beams.dm

```diff
@@ -27,17 +27,29 @@
 	wound_bonus = -30
 	bare_wound_bonus = 40
 
-//overclocked laser, does a bit more damage but has much higher wound power (-0 vs -20)
-/obj/projectile/beam/laser/hellfire
+
+/obj/projectile/beam/laser/hellfire //overclocked laser, does a bit more damage but has much higher wound power (-0 vs -20)
 	name = "hellfire laser"
-	wound_bonus = 0
-	damage = 25
+	wound_bonus = -5
+	damage = 20
```

Comment:
```suggestion
	wound_bonus = 0
	damage = 25
```

Reply:
What bug is this fixing? You have been asked specifically to not use code review to make balance suggestions

Reply:
Shit, they right. I was changing the hellfire bullet back to what it was before but forgot!

Reply:
Just making sure